### PR TITLE
[PT-BR] Update Main_Page

### PR DESCRIPTION
**Pull request by GranularBow8193**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by GranularBow8193** [comment]
@LeoFLT Pode verificar a tradução quando possível?

File: wiki/Main_Page/pt-br.md [hunk 1]
```diff
@@ -1,26 +1,30 @@
 ---
 layout: main_page
-outdated: true
-outdated_since: 1f00c369d836e214e9e2b5a3dc8ea21ff3392217
 ---
 
-<div class="wiki-main-page__blurb">
-Bem-vindo ao osu! wiki, um projeto que contém uma ampla variedade de informação relacionada a osu!.
+<div class="wiki-main-page__blurb">
```
Comment (LeoFLT) [inline wiki/Main_Page/pt-br.md:5]:
```suggestion
<div class="wiki-main-page__blurb">
```

File: wiki/Main_Page/pt-br.md [hunk 2]
```diff
@@ -1,26 +1,30 @@
 ---
 layout: main_page
-outdated: true
-outdated_since: 1f00c369d836e214e9e2b5a3dc8ea21ff3392217
 ---
 
-<div class="wiki-main-page__blurb">
-Bem-vindo ao osu! wiki, um projeto que contém uma ampla variedade de informação relacionada a osu!.
+<div class="wiki-main-page__blurb">   
+Bem-vindo ao osu! wiki, uma base de conhecimento de código abeto que contém uma ampla gama de informação relacionada a osu!, um jogo de ritmo gratuito.
```
Comment (LeoFLT) [inline wiki/Main_Page/pt-br.md:6]:
```suggestion
Bem-vindo ao osu! wiki, uma base de conhecimento de código abeto que contém uma ampla gama de informação relacionada a osu!, um jogo de ritmo gratuito.
```

**Review by LeoFLT — CHANGES_REQUESTED** [review]
Looks good to me otherwise

**Comment by GranularBow8193** [comment]
> Looks good to me otherwise

Done, tell me if something is wrong.

**Review by LeoFLT — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**